packages/cli/extensions/environment.js

```diff
@@ -12,5 +12,4 @@ extendEnvironment((hre) => {
   // Prevent any properties being added to hre.cli
   // other than those defined above.
   Object.preventExtensions(hre.cli);
```

Comment:
If we're not using injected objects, we should consider removing the whole extension thing.

Reply:
Removed